app/helpers/footer_helper.rb

```diff
@@ -1,7 +1,7 @@
 module FooterHelper
   def footer_meta_items
     [
-      { text: t(".guidance"), href: "#" },
+      { text: t(".#{current_service}_guidance"), href: guidance_path },
```

Comment:
I'm a bit hesitant towards naming this "guidance".

Would it make more sense to do something like:

```ruby
def footer_meta_items
  [
    { text: t(".claims.grant_conditions"), href: claims_grant_conditions_path } if current_service == :claims,
    { text: ..., href: ... },
    ...
  ].compact
end
```

and maybe as the next iteration:

```ruby
def footer_meta_items
  {
    claims: claims_footer_meta_items,
    placements: placements_footer_meta_items
  }.fetch current_service
end

def claims_footer_meta_items
  [
    ...
  ]
end
```

Also, the School Placements' guidance link doesn't have content just yet, https://school-placements-beta-4330a9a75ffc.herokuapp.com/guidance. It may end up changing, like ours did.

Reply:
Yes please – let's not lock ourselves in to needing the same page names between each service. There's every chance they'll need different sets of static pages.

Reply:
I like the idea of refactoring this to have claim footer meta items and placements footer meta items.

Let's merge this as it is now and I'll follow this up with another PR. I don't want to do it here as it's a bit of a refactor.